presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/SerializedTaskSource.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.classloader_interface;
+
+import java.io.Serializable;
+
+import static java.util.Objects.requireNonNull;
+
+public class SerializedTaskSource
```

Comment:
nit: maybe call it `SerializedPrestoSparkTaskSource`? -- so when other Presto developers search for `TaskSource`, they don't need to worry about "SerializedTaskSource":

<img width="666" alt="Screen Shot 2020-06-07 at 12 10 39 AM" src="https://user-images.githubusercontent.com/799346/83962583-59c42480-a853-11ea-9d7d-ef84f8528891.png">